internal/pkg/sylog/sylog.go

```diff
@@ -158,6 +157,14 @@ func Debugf(format string, a ...interface{}) {
 // SetLevel explicitly sets the loggerLevel
 func SetLevel(l int) {
 	loggerLevel = messageLevel(l)
+	// the apex log (for umoci) is too noisy at warn level, avoid that
+	if loggerLevel < verbose {
```

Comment:
I'd be happy to take this as a PR into 3.4.2 if the apex loglevel is matched to the sylog level - i.e. hide the apex warnings on `-s` but leave them otherwise. I think we'd need a lot more confidence that umoci was only ever using 'warn' for unimportant things to do what's currently in place in this PR.

Reply:
Alright, I'll make that change.  It will still address the request in #4578.

Reply:
What about the quiet level?  There's no corresponding level in apex.  Can I have that silence apex warnings?

Reply:
I don't think so - sorry for the delay - see below